### ISSUE TITLE
ffmpeg_downloader.installed doesn't check if ffmpeg is installed anywhere else on the system.

### ISSUE THREAD

**Issue by FlorianEagox** [python-ffmpegio/python-ffmpeg-downloader#8]
The `ffmpeg_downloader.installed()` function only seems to check if ffmpeg is installed in the dir this program would install it in, but it doesn't check if it's already in the system path or callable globally from the CLI.

**Comment by tikuma-lsuhsc** [comment]
Right. Because this project was meant to be a cli command, I figured the user would/should first try `ffmpeg` command before `ffdl` to see if the system already has FFmpeg installed. Let me sit on it for a bit.
